apps/css-workshop/src/components/Checkbox.astro

```diff
@@ -0,0 +1,54 @@
+---
+import cx from 'classnames';
+
+type Props = {
+  isLoading?: boolean;
+  label?: React.ReactNode;
+  indeterminate?: boolean;
+  checked?: boolean;
+  disabled?: boolean;
+  variant?: 'eyeball' | 'smile' | 'frown' | 'big frown';
+  content?: React.ReactNode;
+  status?: 'negative' | 'positive' | 'warning' | 'negative';
+} & astroHTML.JSX.HTMLAttributes;
```

Comment:
Chose to do using typing here as the props got a bit more complicated than expected